securedrop_client/gui/widgets.py

```diff
@@ -1896,6 +1921,7 @@ def __init__(
 
         self.controller = controller
         self.file = self.controller.get_file(file_uuid)
+        self.uuid = file_uuid
```

Comment:
can we remove `self.file` since we keep its uuid now?

Reply:
unfortunately we still need this as the initializer calls `_set_file_state` which uses `self.file`